scripts/utils/syncPool.ts

```diff
@@ -0,0 +1,43 @@
+import hre, { ethers, artifacts } from 'hardhat';
+import * as dotenv from 'dotenv';
+import { NamedAddresses } from '../../test/integration/setup/types';
+
+dotenv.config();
+
+const toBN = ethers.BigNumber.from;
+
+// Syncs the uniswap FEI-ETH pair to a price relative to oracle price
+// targetBPs would be multiplied by the peg and divided by 10000 and
+// the pair would sync to that price
+export async function syncPool(targetBPs: number, addresses: NamedAddresses, sendingAddress: string, logging = false) {
+  const feiAddress = addresses.feiAddress;
+  const ethUniswapPCVDepositAddress = addresses.ethUniswapPCVDepositAddress;
+  const ethPairAddress = addresses.ethPairAddress;
+  const fei = await ethers.getContractAt('Fei', feiAddress);
+  const uniswapPcvDeposit = await ethers.getContractAt('UniswapPCVDeposit', ethUniswapPCVDepositAddress);
+  const ethPair = await ethers.getContractAt('IUniswapV2Pair', ethPairAddress);
+
+  logging ? console.log('Current') : undefined;
```

Comment:
nit: we started doing logging && console.log() to remove the `: undefined` if you want to edit

Reply:
Will do!